src/mito2/src/region/opener.rs

```diff
@@ -313,6 +313,10 @@ impl RegionOpener {
         let manifest = manifest_manager.manifest();
         let metadata = manifest.metadata.clone();
 
+        if let Some(ttl) = manifest.ttl {
+            region_options.ttl = Some(ttl);
+        }
```

Comment:
It's better to store the ttl in the table metadata instead of the manifest. cc @v0y4g3r

Reply:
Discuss about this [here](https://github.com/GreptimeTeam/greptimedb/pull/4848#discussion_r1812311692).